### PR TITLE
Document RKE event rate limiting feature

### PR DESCRIPTION
**Pull request by catherineluse**

### CODE REVIEW

File: content/rke/latest/en/config-options/rate-limiting/_index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+For configuration details, refer to the [official Kubernetes documentation.](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#eventratelimit)
+
+### Example Configurations
+
+The following configuration in the `cluster.yml` can be used to enable the event rate limit by default:
+
+```yaml
+services:
+  kube-api:
+    service_cluster_ip_range: 10.44.0.0/24
```
Comment (leodotcloud) [inline content/rke/latest/en/config-options/rate-limiting/_index.md]: `service_cluster_ip_range: 10.44.0.0/24` this line can be removed

File: content/rke/latest/en/config-options/rate-limiting/_index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      type: Server
+...
+```
+
+To customize the event rate limit, the entire Kubernetes resource for the configuration must be provided in the `configuration` directive:
+
+```yaml
+services:
+  kube-api:
+    service_cluster_ip_range: 10.44.0.0/24
```
Comment (leodotcloud) [inline content/rke/latest/en/config-options/rate-limiting/_index.md]: `service_cluster_ip_range: 10.44.0.0/24` this line can be removed.

File: content/rke/latest/en/config-options/rate-limiting/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rke/latest/en/config-options/rate-limiting/_index.md]: removed

File: content/rke/latest/en/config-options/rate-limiting/_index.md (hunk 2 above)
Comment (catherineluse) [reply content/rke/latest/en/config-options/rate-limiting/_index.md]: removed

**Pull request merged**